Remove N25Q512..1G as the model_id is the same as MT25QU512.

diff -Naru ../original/flashrom-v1.2.1/flashchips.c flashrom-v1.2.1/flashchips.c
--- ../original/flashrom-v1.2.1/flashchips.c	2025-10-31 12:09:18.305320542 +0000
+++ flashrom-v1.2.1/flashchips.c	2025-10-31 12:18:46.277505180 +0000
@@ -10858,46 +10858,6 @@
 
 	{
 		.vendor		= "Micron/Numonyx/ST",
-		.name		= "N25Q512..1G", /* ..1G = 1.8V, uniform 64KB/4KB blocks/sectors */
-		.bustype	= BUS_SPI,
-		.manufacture_id	= ST_ID,
-		.model_id	= ST_N25Q512__1G,
-		.total_size	= 65536,
-		.page_size	= 256,
-		/* supports SFDP */
-		/* OTP: 64B total; read 0x4B, write 0x42 */
-		.feature_bits	= FEATURE_WRSR_WREN | FEATURE_OTP | FEATURE_4BA_WREN,
-		.tested		= TEST_UNTESTED,
-		.probe		= probe_spi_rdid,
-		.probe_timing	= TIMING_ZERO,
-		.block_erasers	=
-		{
-			{
-				.eraseblocks = { {4 * 1024, 16384} },
-				.block_erase = spi_block_erase_21,
-			}, {
-				.eraseblocks = { {4 * 1024, 16384} },
-				.block_erase = spi_block_erase_20,
-			}, {
-				.eraseblocks = { {64 * 1024, 1024} },
-				.block_erase = spi_block_erase_dc,
-			}, {
-				.eraseblocks = { {64 * 1024, 1024} },
-				.block_erase = spi_block_erase_d8,
-			}, {
-				.eraseblocks = { {32768 * 1024, 2} },
-				.block_erase = spi_block_erase_c4,
-			}
-		},
-		.printlock	= spi_prettyprint_status_register_n25q, /* TODO: config, lock, flag regs */
-		.unlock		= spi_disable_blockprotect_n25q, /* TODO: per 64kB sector lock registers */
-		.write		= spi_chip_write_256, /* Multi I/O supported */
-		.read		= spi_chip_read, /* Fast read (0x0B) and multi I/O supported */
-		.voltage	= {1700, 2000},
-	},
-
-	{
-		.vendor		= "Micron/Numonyx/ST",
 		.name		= "N25Q512..3G", /* ..3G = 3V, uniform 64KB/4KB blocks/sectors */
 		.bustype	= BUS_SPI,
 		.manufacture_id	= ST_ID,
